src/test/java/org/hisp/dhis/rules/RuleEngineFunctionTests.java

```diff
@@ -790,7 +858,7 @@ public void evaluateD2ZScoreWFHBoy() throws Exception
     public void evaluateD2ZScoreWFHGirl() throws Exception
     {
         RuleAction ruleAction = RuleActionDisplayKeyValuePair.createForFeedback(
-                "test_action_content", "true" );
+                "test_action_content", "d2:zScoreWFH(81.5,3.8,'female') == 2" );
```

Comment:
The content here is not great. A good data value for this rule action would be:
d2:zScoreWFH(81.5,9.6,'female')